### ISSUE TITLE
Zoom-fullscreen should zoom cover the toolbar/simple-bar and padding

### ISSUE THREAD

**Issue by aiguofer** [koekeishiya/yabai#1558]
Currently, it's hard to differentiate between a single window open in a space and multiple windows open in the space but one of them having `zoom-fullscreen` toggled. 

It'd be great if `--toggle zoom-fullscreen` made the window cover the entire display (covering the toolbar and padding). If it's not possible to cover up the native toolbar, it'd be great if at least it could ignore the padding for the `external_bar`.

If modifying the behavior is unwanted by others, it'd be great if there was an alternate `--toggle full-zoom-fullscreen` option.

Basically, I'd love it if it acted like `native-fullscreen` but without creating a new space for it.

**Comment by aiguofer** [comment]
I just noticed today if I double click a window title bar it does just what I described above. Looks like it's a configurable setting [see this post](https://apple.stackexchange.com/a/232153/358126). Wonder if `yabai` has a way to programatically call that behavior?

**Comment by aiguofer** [comment]
Bump

**Comment by albert-ying** [comment]
I think we just need a `--toggle external_bar` that behave like `--toggle gap` and `--toggle padding`

**Comment by aiguofer** [comment]
Ahh that's an interesting thought! yeah I think that would work too. I feel like chaining commands like that make the animation a little clunky, but ultimately I just care about the functionality.

I wish I could go back to my linux laptop running i3.. but alas my company does not support that :(

**Comment by erezsh** [comment]
> Basically, I'd love it if it acted like native-fullscreen but without creating a new space for it.

I would like that too! Does anyone have a solution for it?

**Comment by aspauldingcode** [comment]
Lol this is the same as the request I made about toggling a yabai fullscreen equivilent of native-fullscreen. 

https://github.com/koekeishiya/yabai/issues/2221

**Comment by koekeishiya** [comment]
When a window enters *native fullscreen*, it changes *presentation mode* which causes various changes to the window. This behaviour is not something that can be activated by yabai. It is special to the "native fullscreen-space" version.

I have added a command that ignores all configured padding (both padding and external_bar padding). yabai is not able to move the window into the Notch region or below the macOS menubar, so that space will always be reserved.
It makes no sense to let the window extend into the Dock area either.

If Dock is hidden/has autohide enabled, yabai will extend the window across the Dock space.
If the MacOS menubar is hidden/autohide enabled, yabai will attempt to place the window at the topmost coordinate (gated by the notch on newer macbooks).

This works for both managed and unmanaged windows.

Closing as duplicate of #2221
